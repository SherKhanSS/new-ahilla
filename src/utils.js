export const extend = (a, b) => {
  return Object.assign({}, a, b);
};

export const getCurentFilm = (films, props) => {
  return films.find((movie) => movie.id === Number(props.match.params.id));
};

export const noop = () => {
  // do nothing
};
