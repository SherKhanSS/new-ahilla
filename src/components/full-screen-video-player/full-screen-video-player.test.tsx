import * as React from "react";
import * as renderer from "react-test-renderer";
import FullScreenVideoPlayer from "./full-screen-video-player";
import {noop} from "../../utils";

// const poster: string = `img/bohemian-rhapsody.jpg`;
// const preview: string = `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`;
const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

it(`Render FullScreenVideoPlayer`, () => {
  const tree = renderer
    .create(<FullScreenVideoPlayer
      isPlay={true}
      timeElapsed={0}
      currentProgress={`0`}
      onPlayPauseButtonClick={noop}
      onFullScreenClick={noop}
    ><MockComponent/></FullScreenVideoPlayer>, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
