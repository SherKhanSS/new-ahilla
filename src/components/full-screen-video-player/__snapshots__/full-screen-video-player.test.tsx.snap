// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FullScreenVideoPlayer 1`] = `
<div
  className="player"
>
  <div
    className="visually-hidden"
  >
    <svg
      version="1.1"
      xmlns="http://www.w3.org/2000/svg"
    >
      <symbol
        fill="none"
        height="19"
        id="play-s"
        viewBox="0 0 19 19"
        width="19"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clipRule="evenodd"
          d="M0 0L19 9.5L0 19V0Z"
          fill="#EEE5B5"
          fillRule="evenodd"
        />
      </symbol>
      <symbol
        height="21px"
        version="1.1"
        viewBox="0 0 14 21"
        width="14px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
          id="Artboard"
          stroke="none"
          strokeWidth="1"
        >
          <polygon
            fill="#EEE5B5"
            fillRule="nonzero"
            id="Line"
            points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
          />
          <polygon
            fill="#EEE5B5"
            fillRule="nonzero"
            id="Line"
            points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
          />
        </g>
      </symbol>
    </svg>
  </div>
  <div />
  <button
    className="player__exit"
    onClick={[Function]}
    type="button"
  >
    Exit
  </button>
  <div
    className="player__controls"
  >
    <div
      className="player__controls-row"
    >
      <div
        className="player__time"
      >
        <progress
          className="player__progress"
          max="100"
          value="0"
        />
        <div
          className="player__toggler"
          style={
            Object {
              "left": "0%",
            }
          }
        >
          Toggler
        </div>
      </div>
      <div
        className="player__time-value"
      >
        0:0:0
      </div>
    </div>
    <div
      className="player__controls-row"
    >
      <button
        className="player__play"
        onClick={[Function]}
        type="button"
      >
        <svg
          height="19"
          viewBox="0 0 19 19"
          width="19"
        >
          <use
            xlinkHref="#pause"
          />
        </svg>
        <span>
          Play
        </span>
      </button>
      <div
        className="player__name"
      >
        Transpotting
      </div>
      <button
        className="player__full-screen"
        onClick={[Function]}
        type="button"
      >
        <svg
          height="27"
          viewBox="0 0 27 27"
          width="27"
        >
          <use
            xlinkHref="#full-screen"
          />
        </svg>
        <span>
          Full screen
        </span>
      </button>
    </div>
  </div>
</div>
`;
